templates/shortcode-profile-editor.php

```diff
@@ -28,6 +28,7 @@
 
 	<form id="edd_profile_editor_form" class="edd_form" action="<?php echo edd_get_current_page_url(); ?>" method="post">
 		<fieldset>
+			<?php do_action( 'edd_profile_editor_before_name' ); ?>
```

Comment:
Ok, but I want to add the section "Change your avatar" before "Change your name"

Would I add a new action for this? (Really I do not want to add an excesive number of actions)

and I just saw that the form does not have the attribute enctype, any idea on how to implement this only when necessary?